contracts/croncat-factory/src/contract.rs

```diff
@@ -118,6 +121,43 @@ fn execute_update_config(deps: DepsMut, owner_addr: String) -> Result<Response,
         .add_attribute("owner_addr", config.owner_addr))
 }
 
+fn execute_proxy(
+    deps: DepsMut,
+    info: MessageInfo,
+    msg: WasmMsg,
+) -> Result<Response, ContractError> {
+    let config = &CONFIG
+        .may_load(deps.storage)?
+        .ok_or(ContractError::UnknownContract {})?;
```

Comment:
```suggestion
    let config = CONFIG.load(deps.storage)?;
```
It should load and no need to take a reference here)

Reply:
nice!